packages/terra-doc-template/src/DocTemplate.module.scss

```diff
@@ -22,3 +22,466 @@
     padding-top: var(--terra-doc-template-doc-card-not-empty-padding-top);
   }
 }
+
+:local(.props-table-markdown) {
```

Comment:
I'll work with you to clean this up some, we won't need full markdown styling for the prop descriptions, we should be fine with only a subset of this styling.

Reply:
Looking through this PR again, I think all of this can removed. All the markdown is now running through the terra-markdown component which should be providing all these styles.

Reply:
This is also pointing to an old commit, most of this is already gone. :)

However, I've adjusted the styles include the overall styles not being applied by Markdown, as well as restoring the heading bottom border so that it matches the current doc site.